pyof/v0x01/common/phy_port.py

```diff
@@ -188,7 +188,7 @@ class ListOfPhyPorts(basic_types.FixedTypeList):
     Attributes:
         items (optional): Instance or a list of instances of PhyPort
     """
-    def __init__(self, items=[]):
-        basic_types.FixedTypeList.__init__(self,
-                                           pyof_class=PhyPort,
-                                           items=items)
+    def __init__(self, items=None):
+        super().__init__(self,
```

Comment:
Super is not deprecated. How is deprecated is the self argument you used on
the `super().__init__(self, ...)` Call.

Diego Rabatone Oliveira